### ISSUE TITLE
Google Drive Connector install doc

### ISSUE THREAD

**Issue by ruthaydt** [globusonline/docs.globus.org#175]
These comments are based on the version currently on Staging.       I'm going to add a separate comment for each requested (or pending request) change.

**Comment by ruthaydt** [comment]
Reformat / revise this text - either to what I've suggested or to something else that is clear.   I feel the existing paragraph is too long, and the placement of the "For example, " doesn't quite work with the following 2 sentences.  The first seems to be telling you what to do, and the 2nd seems to be the real example.    My suggestion tries to address these things.

From:
In the [Google Drive] section, set Domains to be a comma separated list of Google domains from which Google Drive can be made accessible via this endpoint. When set, only users who have an account from one of these Google domains can link their Google Drive to this endpoint and make it accessible. For example, if you would like restrict the use of this endpoint such that only users who have an account from your campus Google domain can make their Drive accessible via this endpoint, set the value to be your Google domain. To restrict it to users who have accounts from uchicago.edu or mygoogledomain.com, the value will be set to "uchicago.edu, mygoogledomain.com"

To:
In the [Google Drive] section, set Domains to be a comma separated list of Google domains from which Google Drive can be made accessible via this endpoint. When set, only users who have an account from one of these Google domains can link their Google Drive to this endpoint and make it accessible. 

For example, to restrict this endpoint so that only users who have accounts from the "uchicago.edu" or the "mygoogledomain.edu" domains can use the endpoint to make their Google Drive accessible, Domains would be set to "uchicago.edu, mygoogledomain.edu".

**Comment by ruthaydt** [comment]
I've emailed Vas to see if he want to change the following sentences to be more explicit about the connector being an add-on to the Standard Subscription.   Will update this when I get a response from him:  

The Google Drive connector enables use of Globus services to access and share content from a Google Drive associated with a Google account. The Google Drive connector is a premium feature available only to organizations with a Globus Standard subscription.

**Comment by teresas** [comment]
Instead of creating new issues for any further changes with the Google Drive Connector install doc, comment on this one so that I can make changes on this branch instead of creating more. This way all the changes pertaining to the GDC doc are in the same spot. Currently, we still have 3 other outstanding issues that need review.

**Comment by teresas** [comment]
@ruthaydt @ranantha Can you review #169, #172, and #174 so I can push those to prod before starting on this one?

**Comment by ruthaydt** [comment]
Ok - thanks for the guidance.  I didn’t want to add these to the previous ones because I know it may be awhile before I hear back from Vas, and, I’m mid-review on the others and think what’s currently on staging should be pushed before you tackle any more edits.  Was trying to keep them somewhat separate

**Comment by ruthaydt** [comment]
I’ve reviewed these and found one more change related to 174 — added there.    

I’m not sure if I’m supposed to mark closed, or reviewed, or what the protocol is to indicate I’m done.  Please advise.   Or, perhaps since Rachana also on the reviewer list, you want to wait.

I think the staging version is substantially better than the prod version and recommend pushing it sooner rather than later, even if more updates are needed later.

**Comment by teresas** [comment]
Do not mark anything as closed; I only close issues when they have been fully resolved and the results are up on production. Leave a comment on each issue after things have been reviewed and are good for prod or if any changes still need to be made.

**Comment by ruthaydt** [comment]
Another change for the doc.   

Change: 
    This document describes how to install and use the Google Drive connector.
To:  
    This document describes how to install the Google Drive connector.

**Comment by ruthaydt** [comment]
Re: my pending request to Vas about wording on premium feature - just heard it will be next week before he gets to it.

**Comment by ruthaydt** [comment]
For consistency, use the term "machine", not "system", when you talk about where the software is installed.   (Note that we previously got replaced the term "server" with "machine" in this use case... clear to those fluent in all the concepts/ideas but easier for reader if always consistent usage).   

**Change these three places** to use "machine":

... administrator (root) privileges on your system to install Globus Connect Server; ...

... be set up on your system the first time that the ...

.,.. If your system is behind a firewall, ...

**Comment by danpowers** [comment]
Replacing 'system' with 'machine' seems pretty non-standard. Not what I'm used to seeing in sysadmin/tech doc.

**Comment by ruthaydt** [comment]
Then perhaps we should use system throughout?    Mostly want to use the same word so that it’s clear that what we call “A” in one part of the doc is also called “A” in another part of the doc.  I’m fine with either word as long as it’s consistent.

**Comment by danpowers** [comment]
That might be a good idea. If you find places where we're not being consistent with our terminology then we'd want to fix that.

**Comment by ruthaydt** [comment]
Please disregard my earlier request to change system -> machine, and instead change machine -> system in these eight places:
... will need a separate machine to install Globus Connect Server ...
... The machine will need a publicly resolvable ...
... must be available on the machine. ...
... On the machine where the Globus Connect Server ...
... fully qualified name of the machine where the endpoint...
...  fully qualified host name of the machine where the...
... On the machine where the Globus Connect ...
... resolvable DNS name of the machine. ...

**Comment by ruthaydt** [comment]
9/6 - A couple of the requested changes have not yet been done.   Once they are (or an explanation why they should not be) this issue can be closed.    Added dated edits to original comments to note status as I couldn't think of a better way to indicate status... no comment hierarchy that I could find.

**Comment by teresas** [comment]
The google drive docs have gone through many changes and rewrites over the past few weeks from different people in different forums, making it difficult to follow what changes actually need/needed to be made. I'll add the changes from the last comment 13 days ago (two comments up) and once approved will close this issue to prevent more confusion. Changes beyond that should be discussed with other contributors (@ranantha, @danpowers, not sure of others) and then have a new issue created. I want to make sure you guys are on the same page before an issue is submitted so that these pages aren't constantly having changes.

**Comment by ranantha** [comment]
@teresas agreed, I was just making a similar comment here. 

a) change to "system" rather than machine
b) edit first line to say "This document describes how to install the Google Drive connector."

Please close this out once those changes are made, and we can track others in new issues.
